torch/csrc/distributed/c10d/Store.cpp

```diff
@@ -38,18 +38,34 @@ std::string Store::get_to_str(const std::string& key) {
 }
 
 void Store::append(const std::string& key, const std::vector<uint8_t>& value) {
```

Comment:
This is a fallback when `has_extended_api` is false right?

Reply:
Yup, these fallbacks are in place when the underlying store doesn't implement the newer methods.